src/Razor/src/Microsoft.AspNetCore.Razor.ExternalAccess.RoslynWorkspace/RazorWorkspaceListener.cs

```diff
@@ -11,15 +10,12 @@ public class RazorWorkspaceListener : IDisposable
     private static readonly TimeSpan s_debounceTime = TimeSpan.FromMilliseconds(100);
 
     private string? _projectRazorJsonFileName;
-    private readonly Dictionary<string, TaskDelayScheduler> _workQueues;
+    private readonly Dictionary<ProjectId, TaskDelayScheduler> _workQueues;
```

Comment:
Changed the key here to make testing easier, but also strictly speaking the previous code wouldn't have worked for multi targetting, should that ever be something someone wants to do.